src/atoms/Tooltip/index.js

```diff
@@ -2,22 +2,71 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
 import Icon from 'atoms/Icon';
+import Modal from 'molecules/Modal';
 import styles from './tooltip.module.scss';
 
-function Tooltip( props ) {
-  const {
-    children,
-    className
-  } = props;
-
-  return (
-    <div className={classnames(styles['tooltip-wrapper'], className)}>
-      <Icon icon='questionMark' className={styles['tooltip']} />
-      <div className={styles['hover-message']}>
-        { children }
+/* eslint-disable */
+class Tooltip extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      modalIsOpen: false
+    }
+
+    this.openModal = this.openModal.bind(this);
+    this.closeModal = this.closeModal.bind(this);
+  }
+
+  openModal() {
+    (window.innerWidth <= 768) && this.setState({
+      modalIsOpen: true
+    });
+  }
+
+  closeModal() {
+    this.setState({
+      modalIsOpen: false
+    });
+  }
+
+  render() {
+    const {
+      children,
+      className,
+      left,
```

Comment:
This would be better a stateVariable `direction: left || right`. I'm guessing you don't want to be able to allow all left, center, right conditions?